### PR TITLE
Start title music again if unmuted in title screen

### PR DESCRIPTION
**Pull request by csm10495**
Should close out #16759.

I believe this should be ok since PlayTitleMusic() has checks for if we're in the title screen or in game before actually starting the music.

### CODE REVIEW

**Comment by Gymnasiast** [comment]
Code looks good, could you add a changelog entry?

**Comment by csm10495** [comment]
@Gymnasiast done. If all good, please merge. Thanks!

File: src/openrct2/audio/Audio.cpp [hunk 1]
```diff
@@ -367,6 +367,7 @@ namespace OpenRCT2::Audio
         if (gConfigSound.master_sound_enabled)
         {
             Resume();
+            PlayTitleMusic();
```
Comment (Broxzier) [inline src/openrct2/audio/Audio.cpp:370]: I think it'd be nicer to have this call inside `Resume` instead.

**Review by Broxzier — COMMENTED** [review]
> I believe this should be ok since PlayTitleMusic() has checks for if we're in the title screen or in game before actually starting the music.

Sounds like this is best renamed to better reflect what it does. That can be left for a future PR, though.

**Comment by csm10495** [comment]
> > I believe this should be ok since PlayTitleMusic() has checks for if we're in the title screen or in game before actually starting the music.
> 
> Sounds like this is best renamed to better reflect what it does. That can be left for a future PR, though.

The reason it's like this is to match the behavior a few lines down to mute.

Should pause also stop title music to match up?

File: distribution/changelog.txt [hunk 2]
```diff
@@ -64,6 +64,7 @@
 - Fix: [#16591] [Plugin] setInterval and setTimeout is not disposed when map unloads.
 - Fix: [#16711] [Plugin] Car.rideObject overflowing with more than 256 ride types.
 - Fix: [objects#165] Glitch when Bengal Tiger Cars go through a corner.
+- Fix: [#16779] Fix case where title music doesn't unmute properly.
```
Comment (Gymnasiast) [inline distribution/changelog.txt:67]: This should be above the previous entry.

**Comment by Broxzier** [comment]
@csm10495 You can leave it like this, it's more of a refactoring thing rather than a fix. Please address the changelog note by @Gymnasiast and then this can get merged.

**Comment by Gymnasiast** [comment]
> @Gymnasiast done. If all good, please merge. Thanks!

@csm10495 If it’s all good, we _will_ merge a PRs. There is no need to rush us, it comes across as quite impatient. (Reminding us when we don’t respond for a week is fine, obviously.)

**Pull request merged**